src/Surfnet/StepupRa/RaBundle/Service/RaSecondFactorExport.php

```diff
@@ -18,38 +18,50 @@
 
 namespace Surfnet\StepupRa\RaBundle\Service;
 
-use EE\DataExporterBundle\Service\DataExporter;
 use Psr\Log\LoggerInterface;
 use Surfnet\StepupMiddlewareClientBundle\Identity\Dto\RaSecondFactorExportCollection;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\StreamedResponse;
 
 class RaSecondFactorExport
 {
-    /**
-     * @var DataExporter
-     */
-    private $exporter;
-
```

Comment:
Removing the EE Data Exporter is fine with me. I however worry about regressions. Sadly I did not add tests while writing this feature. That makes this change more dangerous in my opinion. 

What was the reason for moving away from using the EE Exporter service?

Reply:
I've added a test in deploy earlier on and I've used that, only a bit more quoting is now done but the result is the same:

https://github.com/OpenConext/Stepup-Deploy/commit/c6e741f7d275f009edeec166a455dfd0dafa4326#diff-5ca340d019007b0ab97b4b9b9c52a5d8